app/views/users/show.html.erb

```diff
@@ -1,29 +1,35 @@
 <%= title('Dashboard') %>
 
 <div class="container tmp-container-spacer">
```

Comment:
Container is from bootstrap. but we have our own page-container that will keep padding the same across the site 
```suggestion
<div class="page-container">
```